db/datastruct/existence/existence_filter.go

```diff
@@ -109,8 +109,7 @@ func (b *Filter) Build() error {
 	}
 
 	log.Trace("[agg] write file", "file", b.FileName)
-	tmpFilePath := b.FilePath + ".tmp"
-	cf, err := os.Create(tmpFilePath)
+	cf, err := dir.CreateTemp(b.FilePath)
```

Comment:
Please make sure you don’t loosing .tmp extension. Because erigon does rm them at startup